civihr_default_theme/scss/components/_leave-request-modal.scss

```diff
@@ -0,0 +1,80 @@
+$row-margin-bottom: 15px;
+
+.chr_leave-request-modal__user-info {
+  padding: 20px 20px;
+  font-size: $crm-font-size-large;
+}
+
+.chr_leave-request-modal__response {
+  padding: 20px 20px 0;
+}
+
+.chr_leave-request-modal__response-label {
+  font-size: $crm-font-size-large;
+  font-weight: $crm-font-weight-regular;
+  margin-top: 4px;
+
+  span {
+    color: $brand-danger;
+  }
+}
+
+.chr_leave-request-modal__response-dropdown {
+  padding-right: 20px;
+}
+
+.chr_leave-request-modal__comment {
+  border-bottom: none !important;
+  margin: 15px 25px 0;
+  width: calc(100% - 50px);
+}
+
+.chr_leave-request-modal__add-comment {
+  background-color: $light;
+  border: 1px solid $gray-light;
+  border-top: none;
+  margin: 0 25px;
+  padding: 5px 0;
+  text-align: right;
+  width: calc(100% - 50px);
+
+  hr {
+    margin: 0 10px;
+  }
+
+  button.btn-link {
+    margin-top: 5px;
+  }
+
+  i {
```

Comment:
Reference the `.fa` class, not the `i` element, otherwise you're making this style markup-dependent